deploy/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -454,12 +454,6 @@ rules:
   - apiGroups: [""]
     resources: ["events"]
     verbs: ["list", "watch", "create", "update", "patch"]
-  - apiGroups: ["storage.k8s.io"]
-    resources: ["volumeattachments"]
-    verbs: ["get", "list", "watch", "patch"]
-  - apiGroups: ["storage.k8s.io"]
-    resources: ["volumeattachments/status"]
-    verbs: ["patch"]
```

Comment:
Removing this global settings could cause issues for RBD deployment @yati1998

Reply:
This RBAC is specific to cephfs and will not cause any issue to RBD.

Reply:
Yeah.. noticed that and was discussing the same with @yati1998 to ignore above.